application/templates/grafanadashboard.yaml

```diff
@@ -0,0 +1,28 @@
+
+{{- if .Values.grafanaDashboard -}}
+{{- if .Values.grafanaDashboard.enabled -}}
+{{- range $name, $content := .Values.grafanaDashboard.contents }}
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  labels:
+    grafanaDashboard: grafana-operator
```

Comment:
why is this label static?

Reply:
This label is used for grafana operator to select GrafanaDashboard. It will be ignored without this label.

Reply:
yes but shouldn't we take it from values? does it mean same all the time or if we change selector in operator, it gets changed?

Reply:
We don't want to change it in operator. If we have to, we have to change the chart template as well. But it's better be transparent for the user of the chart.

Reply:
OK